core/app/models/spree/order.rb

```diff
@@ -73,10 +73,9 @@ class CannotRebuildShipments < StandardError; end
     # Customer info
     belongs_to :user, class_name: Spree::UserClassHandle.new, optional: true
     belongs_to :bill_address, foreign_key: :bill_address_id, class_name: 'Spree::Address', optional: true
-    alias_attribute :billing_address, :bill_address
-
+    alias_method :billing_address, :bill_address
     belongs_to :ship_address, foreign_key: :ship_address_id, class_name: 'Spree::Address', optional: true
-    alias_attribute :shipping_address, :ship_address
+    alias_method :billing_address, :bill_address
```

Comment:
I think this line should read `alias_method shipping_address, :ship_address`, as currently it's the same as the changes from line 76 🤔